### PR TITLE
LE parser applies relocations on load

### PR DESCRIPTION
**Pull request by user7**

### CODE REVIEW

File: src/nc/input/le/LeParser.cpp [hunk 1]
```diff
@@ -335,8 +334,21 @@ void LeParser::doParse(QIODevice *in, core::image::Image *image, const LogToken
             image->addRelocation(std::make_unique<core::image::Relocation>(
                     page_virt_addr + fh.srcoff,   // relocation virtual address
                     image->symbols()[dstobj - 1], // section alias as base
-                    width,                        // 4 byte relocations only
+                    width,
                     dst));                        // offset from section start
+
+            if (width == 4) {
+                uint32_t dst_virt = dst + image->sections()[dstobj - 1]->addr();
+                ByteOrder::convert(&dst_virt, width, ByteOrder::Current, bo);
```
Comment (yegord) [inline src/nc/input/le/LeParser.cpp]: I guess, you can just write `po.convertTo(dst_virt)`.

Comment (user7) [reply src/nc/input/le/LeParser.cpp]: There's no method convertTo in that class, only convertFrom. Is adding convertTo better than my usage?

Comment (yegord) [reply src/nc/input/le/LeParser.cpp]: I guess, it is absent, because it was never needed.
Could you add it?

Comment (user7) [reply src/nc/input/le/LeParser.cpp]: sure

**Comment by user7** [comment]
Did you see my commit? Or am I supposed to create new pull request? Don't have much experience with github.

**Comment by yegord** [comment]
Yep. I do not have the fastest CI in the world, it does not build a dozen builds instantly. :) You do not need to do anything.

**Comment by yegord** [comment]
Merged. Thanks!

**Pull request closed without merging**